### DISCUSSION TITLE
Not all team members can delete a discussion

### DISCUSSION THREAD

**Discussion by hawkeye64** [community/community#3772] [Discussions]
Out of all the team members in our organization, there are only two that can delete a discussion. One is the creator of the repo, the second has the same permissions as the others (that we can see).

Here is the repo and org: https://github.com/quasarframework/quasar

Are we missing something perhaps?

**Comment by hawkeye64** [comment]
Nothing?

**Reply by hawkeye64** [reply]
Seems like it recently got fixed...

**Comment by laymonage** [comment]
Yeah, it's likely that it was fixed as part of https://github.com/github/feedback/discussions/2937#discussioncomment-818904.
